src/open_inwoner/js/components/readmore/index.js

```diff
@@ -8,7 +8,6 @@ export class ReadMore {
   }
 
   toggleReadMore(event) {
-    event.preventDefault()
```

Comment:
@alextreme this change was committed three months ago, so this must have been broken for a while?

Reply:
Readmore was added in v1.11, so this would make sense